autoware_plugin/src/autoware_plugin.cpp

```diff
@@ -31,11 +32,17 @@ namespace autoware_plugin
     {
         nh_.reset(new ros::CARMANodeHandle());
         pnh_.reset(new ros::CARMANodeHandle("~"));
-        trajectory_pub_ = nh_->advertise<cav_msgs::TrajectoryPlan>("plan_trajectory", 1);
         
+        // trajectory_pub_ = nh_->advertise<cav_msgs::TrajectoryPlan>("plan_trajectory", 1);
+
+        maneuver_srv_ = nh_->advertiseService("/plugins/AutowarePlugin/strategic_plan/plan_maneuvers", &AutowarePlugin::plan_maneuver_cb, this);
+
+        trajectory_srv_ = nh_->advertiseService("/plugins/AutowarePlugin/plan_trajectory", &AutowarePlugin::plan_trajectory_cb, this);
+
+                
         autoware_plugin_discovery_pub_ = nh_->advertise<cav_msgs::Plugin>("plugin_discovery", 1);
         cav_msgs::Plugin msg;
-        msg.name = "Autoware Plugin";
+        msg.name = "AutowarePlugin";
```

Comment:
Is there a reason the name changed?

Reply:
@Supraluminal  said there should not be a space in the name.